ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
```

Comment:
split the css file, into 2 sections:

1- first section are the globals, which means, everything that is generic goes here: In example

a { text-decoration: none } = this is a global setup because you want to add it to every anchor